### PR TITLE
Update release note

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

File: release-notes-1.0.0.md [hunk 1]
```diff
@@ -127,19 +127,19 @@ Java interoperability is a key feature in jBallerina that allows you to call Jav
...
 - Introduce capability to engage custom providers and handlers for inbound/outbound authentication.
 - Introduce OAuth2 inbound authentication.
 - Introduce own modules for different authentication mechanisms (JWT, LDAP, OAuth2 etc.).
 - Introduce prior knowledge support to the HTTP/2 client.
 - Add flow control support to HTTP/2 client and server.
 - Introduce XSLT transformation support.
-- `h2` and `mysql` database client modules and the `sql` module have been discontinued. The `ballerinax/java.jdbc` client module can be used to interact with relational databases.
-- The ByteChannel read API was updated to return only `byte[]|io:Error`, removing the previously returned read byte length.
+- `ballerina/h2` and `ballerina/mysql` database client modules and the `ballerina/sql` module have been discontinued. The `ballerinax/java.jdbc` client module can be used to interact with relational databases.
+- The `ByteChannel` read API was updated to return only `byte[]|io:Error`, removing the previously returned read byte length.
```
Comment (pubudu91) [inline release-notes-1.0.0.md]: This sentence still sounds weird. Any ideas on how this can be improved? @MaryamZi

File: release-notes-1.0.0.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Ballerina 1.0.0 consists of improvements to the language syntax and semantics ba
   - `ballerina/lang.float` for basic type `float`
   - `ballerina/lang.future` for basic type `future`
   - `ballerina/lang.int` for basic type `int`
-  - `ballerina/lang.map` for basic type `mapping`
+  - `ballerina/lang.map` for basic type mapping
```
Comment (MaryamZi) [inline release-notes-1.0.0.md]:
```suggestion
  - `ballerina/lang.map` for mapping types
```

File: release-notes-1.0.0.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Ballerina 1.0.0 consists of improvements to the language syntax and semantics ba
 
 ## Language
 
-- A set of modules, which contain functions associated with the basic types have been introduced. These modules are referred to as the lang library. Each basic type has a corresponding lang library module. Additionally, there is also the `lang.value` module, which holds functions common for all the types. The following is the list of lang library modules.
+- A set of modules, which contain functions associated with the basic types have been introduced. Collectively, these modules are referred to as the lang library. Each basic type has a corresponding lang library module. Additionally, there is also the `lang.value` module, which holds functions common for all the types. The following is the list of lang library modules.
 
   - `ballerina/lang.value`
   - `ballerina/lang.array` for basic type list
```
Comment (MaryamZi) [inline release-notes-1.0.0.md]:
```suggestion
  - `ballerina/lang.array` for list types
```

File: release-notes-1.0.0.md (hunk 1 above)
Comment (MaryamZi) [reply release-notes-1.0.0.md]:
```suggestion
- The byte channel read API was updated to return only `byte[]|io:Error`.
```

**Pull request merged**